### PR TITLE
Devex 922: separate app context param in lambdas - documents pt 3

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: shared/src/business/useCases/externalDocument/fileExternalDocumentForConsolidatedInteractor.js [hunk 1]
```diff
@@ -15,13 +15,24 @@ const { pick } = require('lodash');
...
-  docketNumbersForFiling,
-  documentMetadata,
-  leadDocketNumber,
-  //filingPartyNames? filingPartyMap?,
-}) => {
+  {
+    docketNumbersForFiling,
+    documentMetadata,
+    leadDocketNumber,
+    //filingPartyNames? filingPartyMap?,
```
Comment (matthopson) [inline shared/src/business/useCases/externalDocument/fileExternalDocumentForConsolidatedInteractor.js]: Do we need this?

Comment (sutt0n) [reply shared/src/business/useCases/externalDocument/fileExternalDocumentForConsolidatedInteractor.js]: looks like we don't. i'll remove it.

File: web-api/src/documents/saveSignedDocumentLambda.js [hunk 2]
```diff
@@ -13,10 +13,11 @@ exports.saveSignedDocumentLambda = event =>
...
-      docketNumber,
-      originalDocketEntryId,
-      ...JSON.parse(body),
-    });
+    return await applicationContext
+      .getUseCases()
+      .saveSignedDocumentInteractor(applicationContext, {
+        docketNumber,
+        originalDocketEntryId,
+        ...JSON.parse(body),
```
Comment (matthopson) [inline web-api/src/documents/saveSignedDocumentLambda.js]: Should we move this up, so the body params can never overwrite docketNumber and originalDocketEntryId?

**Review by matthopson — APPROVED** [review]
Couple small nits, and 👍

**Pull request merged**